### ISSUE TITLE
Nora service offline?

### ISSUE THREAD

**Issue by virol** [andrei-tatar/nora-service#44]
Is it possible that the Nora service is offline for the moment?
also Nora login site https://node-red-google-home.herokuapp.com/login  gives me application error

thx

**Comment by andrei-tatar** [comment]
I noticed it this morning. Will look through the logs.

**Comment by virol** [comment]
thx Andrei

----- Original Message -----
From: Andrei Tatar ***@***.***>
To: andrei-tatar/nora-service ***@***.***>
Cc: virol ***@***.***>,  Author ***@***.***>
Date: Wed, 14 Apr 2021 23:34:27 -0700
Subject: Re: [andrei-tatar/nora-service] Nora service offline? (#44)

**Comment by andrei-tatar** [comment]
Unfortunately, not much I can do. 
There are way too many connections attempts for the heroku instance to handle.

I would recommend switching to https://smart-nora.eu/ (https://flows.nodered.org/node/node-red-contrib-smartnora)
https://assistant.google.com/services/a/uid/000000ce9a75a103?hl=en
It's been approved by Google so you should be able to link in the next few days without joining the alpha testers.
This new version will have the ability to handle tens of thousands of connections

```
H11 - Backlog too deep
When the backlog on a particular router passes a threshold, the router determines that 
your application isn't keeping up with its incoming request volume. You'll see an H11
error for each incoming request as long as the backlog is over this size.
```

**Comment by virol** [comment]
Ok will take a look to smart-nora, thx anyway for this service

----- Original Message -----
From: Andrei Tatar ***@***.***>
To: andrei-tatar/nora-service ***@***.***>
Cc: virol ***@***.***>,  Author ***@***.***>
Date: Wed, 14 Apr 2021 23:43:38 -0700
Subject: Re: [andrei-tatar/nora-service] Nora service offline? (#44)

**Comment by virol** [comment]
I see that you are working on smartnora Version 1.0.0 Bèta 1, is it a matter of day's before publish this first non-alfa version on node-red, or takes it longer time and prefer you to use the alfa 6 version (if so how do i use the alfaversion)
many thanks

**Comment by andrei-tatar** [comment]
@virol yes, the latest version is `1.0.0-beta.1`. not sure how often node-red updates the node palette index. It probably takes a few days.
I think you should always use the latest until we have a stable released version.
